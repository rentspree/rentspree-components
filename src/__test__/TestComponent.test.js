import React from 'react'
import TestComponent from '../components/form'
import renderer from 'react-test-renderer'

test('Test component showed', () => {
  console.log(TestComponent)
  const component = renderer.create(
    <TestComponent />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
