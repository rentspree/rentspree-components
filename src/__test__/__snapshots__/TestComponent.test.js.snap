// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component showed 1`] = `
<div
  className="testClass"
>
  Hello world!! This is Foo Bar.
</div>
`;
